docs/pages/sections.js

```diff
@@ -1,16 +1,16 @@
-const sections = [
-  {
-    name: 'Instalação e utilização',
-    content: 'pages/installation.md',
-  },
-  {
-    name: 'Tipografia e cores',
-    content: 'pages/thypography-colors.md',
-  },
-  {
-    name: 'Documentação',
-    components: '../src/components/*.vue',
-  },
+const sections = [{
+  name: 'instalacao_e_utilizacao',
```

Comment:
why this change? 🤔

Reply:
Styleguidist cant deal with spaces or special characters in the title.